src/command_modules/azure-cli-keyvault/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.2.10
+++++++
+* Minor fixes
```

Comment:
Since this is a bug fix, it should describe the bug being fixed:
`* `keyvault secret backup`: Fixes issue where some users received an 'unexpected_keyword' error when using --id.`